Ludown/lib/parseFileContents.js

```diff
@@ -385,15 +373,18 @@ const mergeResults_closedlists = function(blob, finalCollection, type) {
 const parseAndHandleEntity = function(parsedContent, chunkSplitByLine, locale, log) {
     // we have an entity definition
     let entityDef = chunkSplitByLine[0].replace(PARSERCONSTS.ENTITY, '').split(':');
-    let entityName = entityDef[0];
-    let entityType = entityDef[1];
+    let entityName = entityDef[0].trim();
+    let entityType = entityDef[1].trim();
+    let entityRoles = [];
+    let pEntityName = (entityName === 'PREBUILT')?entityType:entityName;
     // see if we already have this as Pattern.Any entity
-    // see if we already have this in patternAny entity collection; if so, remove it
+    // see if we already have this in patternAny entity collection; if so, remove it but remember the roles (if any)
     for(let i in parsedContent.LUISJsonStructure.patternAnyEntities) {
-        if(parsedContent.LUISJsonStructure.patternAnyEntities[i].name === entityName) {
+        if(parsedContent.LUISJsonStructure.patternAnyEntities[i].name === pEntityName) {
             if(entityType.toLowerCase().trim().indexOf('phraselist') === 0) {
```

Comment:
unrelated to this changeset, you can optionally use `entityType.toLowerCase().trim().includes('phraselist')` that returns a `boolean`.